common/src/main/java/io/druid/timeline/TimelineObjectHolder.java

```diff
@@ -28,16 +28,19 @@
 {
   private final Interval interval;
   private final VersionType version;
+  private final long approxSize;
   private final PartitionHolder<ObjectType> object;
 
   public TimelineObjectHolder(
       Interval interval,
       VersionType version,
+      long approxSize,
```

Comment:
do we have to maintain this value as part of the timeline object? Memory pressure is already very high on the brokers from maintaining the serverview and timeline. Is there a way we could get this information when requested as opposed to storing it all the time?